project/api/tests.py

```diff
@@ -313,31 +313,31 @@ def test_api_can_create_territory(self):
         data = {
             "start_date": "0006-01-01",
             "end_date": "0007-01-01",
-            "nation": 1,
+            "nation": self.new_nation.id,
             "references": ["https://en.wikipedia.org/wiki/Test"],
             "geo": '{"type": "MultiPolygon","coordinates": [[[ [102.0, 2.0], [103.0, 2.0], [103.0, 3.0], [102.0, 3.0], [102.0, 2.0] ]],[[ [100.0, 0.0], [101.0, 0.0], [101.0, 1.0], [100.0, 1.0], [100.0, 0.0] ],[ [100.2, 0.2], [100.8, 0.2], [100.8, 0.8], [100.2, 0.8], [100.2, 0.2] ]]]}',
         }
         self.client.credentials(HTTP_AUTHORIZATION="Bearer " + getUserToken())
         response = self.client.post(url, data, format="json")
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self.assertEqual(Territory.objects.count(), 3)
-        self.assertEqual(Territory.objects.last().nation, Nation.objects.get(pk=1))
+        self.assertEqual(Territory.objects.last().nation, self.new_nation)
 
     def test_api_can_update_territory(self):
         """
         Ensure we can update individual territories
         """
-        url = reverse("territory-detail", args=[1])
+        url = reverse("territory-detail", args=[self.territory.id])
         data = {
             "start_date": "0010-01-01",
             "end_date": "0011-01-01",
-            "nation": 1,
+            "nation": self.child_nation.id,
```

Comment:
Why do we have a child_nation attribute?

Reply:
`self.child_nation` refers to the `cls.child_nation` we set up in the `setUpTestData` method at the top of this class.